src/completionProvider.ts

```diff
@@ -1,11 +1,58 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
+import * as fg from "fast-glob";
+import * as _ from "lodash";
+import * as os from "os";
+import * as path from "path";
 import * as vscode from "vscode";
 import Lexx from "xml-zero-lexer";
 
+type Artifact = {
+    groupId: string;
+    artifactId: string;
+    version: string;
+    pom: string;
+};

```

Comment:
Add an empty line here

Reply:
will remove `Artifact` because we only need to save a list of versions here.